File: Breakfast/Java/IntegerReverse.java
public class IntegerReverse {
    public static void main(String[] args) {
        int number = 12345678;
        int reverse = 0;
        
     
        for (int i = 0; number > 0; number /= 10) {
            int digit = number % 10;
            reverse = reverse * 10 + digit;
        }
        
        System.out.println(reverse);
    }
}

File: Breakfast/JavaScript/IntegerReverse.js
 let number = 12345678;
 let reverse = 0;
        
     
        for (let i = 0; number > 0; number /= 10) {
            let digit = number % 10;
            reverse = reverse * 10 + digit;
        }
        
        console.log(reverse);
